diff --git a/node_modules/expo-screen-orientation/ios/ScreenOrientationReactDelegateHandler.swift b/node_modules/expo-screen-orientation/ios/ScreenOrientationReactDelegateHandler.swift
index 287a2a5..86144da 100644
--- a/node_modules/expo-screen-orientation/ios/ScreenOrientationReactDelegateHandler.swift
+++ b/node_modules/expo-screen-orientation/ios/ScreenOrientationReactDelegateHandler.swift
@@ -4,7 +4,7 @@ import ExpoModulesCore
 
 class ScreenOrientationReactDelegateHandler: ExpoReactDelegateHandler {
   // swiftlint:disable-next overridden_super_call
-  override func createRootViewController() -> UIViewController {
+  func createRootViewController() -> UIViewController {
     return ScreenOrientationViewController(defaultScreenOrientationProvider: {
       return self.getAndExportScreenOrientation()
     })

</rewritten_file> 